### PR TITLE
fix(Select): id property

### PR DESCRIPTION
**Pull request by GermanVor**

### CODE REVIEW

File: src/components/Select/Select.tsx [hunk 1]
```diff
@@ -316,6 +316,7 @@ export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(function
             className={selectBlock(mods, className)}
             {...focusWithinProps}
             style={inlineStyles}
+            id={selectId}
```
[the remark this answers is not in the document]
Comment (GermanVor) [reply src/components/Select/Select.tsx]: https://github.com/gravity-ui/uikit/pull/1604/commits/d8a23400f257ed3ac9135da443401517ae5ca9d8

File: src/components/Select/Select.tsx [hunk 2]
```diff
@@ -283,6 +282,11 @@ export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(function
         return null;
     };
 
+    const uniqId = useUniqId();
+    const controlId = id ?? uniqId;
+    const selectId = `select-${controlId}`;
```
Comment (korvin89) [inline src/components/Select/Select.tsx:287]: Do we really need both ids `selectId` and `controlId`?

Comment (GermanVor) [reply src/components/Select/Select.tsx:287]: Honestly, I do not know.

Seems like `SelectControlProps.selectId` was about only `aria-activedescendant` not `id` attribute. 
Also it is required property, while native `id` attribute is optional.

File: src/components/TreeSelect/TreeSelect.tsx [hunk 3]
```diff
@@ -231,7 +233,7 @@ export const TreeSelect = React.forwardRef(function TreeSelect<T>(
                 handleClose={handleClose}
                 disablePortal={popupDisablePortal}
                 mobile={mobile}
-                id={`tree-select-popup-${treeSelectId}`}
+                id={popupId}
```
Comment (korvin89) [inline src/components/TreeSelect/TreeSelect.tsx:236]: We don't use this property, it could be deleted

**Review by korvin89 — APPROVED** [review]

**Pull request merged**